raiden-dapp/src/components/notification-panel/NotificationCard.vue

```diff
@@ -96,14 +96,21 @@ export default class NotificationCard extends Vue {
       flex-direction: column;
       margin-left: 16px;
 
+      &__block-count,
+      &__received {
+        font-size: 12px;
+      }
+
       &__title {
         color: $primary-color;
       }
 
-      &__received,
       &__block-count {
+        color: $color-white;
```

Comment:
Please consider to take advantage of Vueitfy and apply `white--text` to the template directly. This prevents us from doing this manually over and over again and we will still not produce a mass of classes nobody how to use them or when to delete what.
But this is just a general recommendation to use the styling here only for more special stuff. Vuetify comes with a ton of convenient helpers that leave our code more clean.

Reply:
Yeah, I know. But since it is already defined and we are mostly using the colors in the styles I'll just leave it as is. Also, I think it is a bit nice to know that I only need to look in the styles to find styles related code.